source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak.md

```diff
@@ -191,70 +252,84 @@ with ports used by another application or if you use a load balancer or
 proxy server.
 </div>
 
-It is also recommended that you increase the size of Riak's `pb_backlog`
-to be greater than the size of `request_pool` specified in the Riak CS
-`app.config` file. At minimum, `pb_backlog` should be set to `64`. The
-default value is `5`. To do so, uncomment the appropriate line and
-replace `{pb_backlog, 5}` with this:
-
-```appconfig
-{riak_api, [
-            %% Other configs
-            {pb_backlog, 64},
-            %% Other configs
-           ]}
-```
+It is also recommended that users insure that the size of Riak's
+`protobuf.backlog` (or in the `advanced.config`/`app.config` files, the
+`pb_backlog`) is equal to or greater than the size of the
+`pool.request.size`, specified in the Riak CS `riak-cs.conf` (or
+the `request_pool` size in the `advanced.config`/`app.config` files).
 
-If the `request_pool` value in Riak CS is changed, the `pb_backlog`
+If the `pool.request.size` value in Riak CS is changed, the `protobuf.backlog`
 value in Riak should be updated as well.
 
 ## Other Riak Settings
 
-The `app.config` file includes other settings, such as turning on the
-creation of log files and specifying where to store them. These
+The `riak.conf` and `advanced.config` files includes other settings, such as
+turning on the creation of log files and specifying where to store them. These
 settings have default values that should work in most cases. For more
-information, we recommend reading our [configuration files](:w
-) doc
+information, we recommend reading our [configuration files](:w) documentation.
```

Comment:
Dude, I don't know. It was there before, so I'm kinda scared to touch it.
I'll look into it now, though, as my brain is more alive than it used to be,